proxy/InkAPI.cc

```diff
@@ -2208,6 +2208,20 @@ TSUrlPathSet(TSMBuffer bufp, TSMLoc obj, const char *value, int length)
   return URLPartSet(bufp, obj, value, length, &URL::path_set);
 }
 
+void
+TSUrlRemoveHostName(TSMBuffer bufp, TSMLoc obj)
```

Comment:
Sorry for my ignorance. I am new to ATS and am learning. Could you please be more specific? Do I need to add test cases to test the new API?